NameGame/NameGame/UI/Menu/MenuWorkflow.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  MenuWorkflow.swift
+//  NameGame
+//
+//  Created by Tammy Le on 6/23/17.
+//  Copyright © 2017 WillowTree Apps. All rights reserved.
+//
+
+import Foundation
+import ReSwift
+
+protocol CoordinatorHandler {
```

Comment:
Let's change this protocol to a `: class` type. Also possibly rename to `MenuWorkflowPresentationEventObserver`.